recipes-kernel/kexec/kexec-tools_2.0.21.bbappend

```diff
@@ -0,0 +1,11 @@
+SUMMARY = "nilrt specific modification for kdump"
+FILESEXTRAPATHS_prepend := "${THISDIR}/kexec-tools:"
+
+SRC_URI += " file://kdump.conf \
+           "
+
+pkg_postinst_kdump() {
+#!/bin/sh
```

Comment:
I don't think this shebang line is used at all

Reply:
Removed.